posthog/src/main/java/com/posthog/PostHogConfig.kt

```diff
@@ -102,6 +103,14 @@ public open class PostHogConfig(
      * generating anonymous id (which as of now is just random UUID v4)
      */
     public var getAnonymousId: ((UUID) -> UUID) = { it },
+    /**
+     * Determines the behavior for processing user profiles.
+     * - `"always"`: Always processes user profile data for all events.
+     * - `"never"`: Never processes user profile data. This means that anonymous users will not be merged when they sign up or log in.
+     * - `"identified_only"` (default): Processes user profiles only when an identification event occurs, such as `identify`, `alias`, or `groupidentify`.
```

Comment:
```suggestion
     * - `"identified_only"` (default): Processes user profiles only when an identification event occurs, such as `identify`, `alias`, or `groupidentify`.
```

this is not up to date since the `$process_person_profile` is added for all events

Reply:
> this is not up to date since the `$process_person_profile` is added for all events

sorry, I don't know if I understand these commit suggestion

Reply:
```suggestion
     * - `"identified_only"` (default): Processes user profiles for all events, with additional behavior when an identification event occurs, such as `identify`, `alias`, or `groupidentify`.
```

what do you think?